### PR TITLE
exposing ROI in strategy card

### PR DESCRIPTION
**Pull request by zavelevsky**

### CODE REVIEW

**Comment by ashachaf** [comment]
- [ ] update tool tip text
current: Total returns of the strategy from the creation. 
updated: Total percentage returns of the strategy from its creation.

- [ ] decimals
current: we show 1 decimal number (i.e. 11.2%)
updated: show 2 decimals (i.e. 11.23%)

**Review by pingustar — APPROVED** [review]
Code LGTM! 

Added some small nit comments

File: src/libs/modals/modals/ModalManageNotifications.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  strategyId: string;
+};
+
+export const ModalManageNotifications: ModalFC<
+  ModalManageNotificationsData
+> = ({ id, data: { strategyId } }) => {
+  const { closeModal } = useModal();
+  const onClick = () => {
+    window?.open(
+      `https://app.hal.xyz/recipes/carbon-track-strategy-updated?strategy_id=${strategyId}`,
```
[the remark this answers is not in the document]
Comment (zavelevsky) [reply src/libs/modals/modals/ModalManageNotifications.tsx:18]: This is very specific to this component and also requires knowledge of the URL structure (as it adds a param) - so I don't see a benefit in moving it there

File: src/components/strategies/overview/strategyBlock/StrategyBlockRoi.tsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+import { Link } from 'libs/routing';
+import { ReactComponent as IconLink } from 'assets/icons/link.svg';
+import { ReactComponent as IconTooltip } from 'assets/icons/tooltip.svg';
+
+export interface StrategyBlockRoiProps {
+  strategyId: string;
+}
+
+export const StrategyBlockRoi: FC<StrategyBlockRoiProps> = ({ strategyId }) => {
+  const { strategyRoi } = useRoi(strategyId);
```
[the remark this answers is not in the document]
Comment (zavelevsky) [reply src/components/strategies/overview/strategyBlock/StrategyBlockRoi.tsx:15]: > no loading animation / error state needed?

@Costa769 - do we have a spec for loading animation while ROI isn't loaded it?

File: src/utils/carbonApi.ts [hunk 3]
```diff
@@ -36,6 +59,12 @@ const carbonApi = {
     );
     return data;
   },
+  getRoi: async (): Promise<RoiRow[]> => {
+    const {
+      data: { data },
+    } = await carbonApiAxios.get<{ data: RoiRow[] }>('roi');
```
[the remark this answers is not in the document]
Comment (zavelevsky) [reply src/utils/carbonApi.ts:65]: yes

File: src/components/strategies/overview/strategyBlock/StrategyBlockRoi.tsx [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+        <IconTooltip className="h-10 w-10" />
+      </Tooltip>
+    </div>
+    <div className={`text-24 ${negative ? 'text-red' : 'text-green'}`}>
+      {value}
+    </div>
+  </div>
+);
+
+const formatNumber = (num: BigNumber): { value: string; negative: boolean } => {
```
[the remark this answers is not in the document]
Comment (zavelevsky) [reply src/components/strategies/overview/strategyBlock/StrategyBlockRoi.tsx]: done, plus tests

File: src/hooks/useRoi.ts [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import BigNumber from 'bignumber.js';
+import { useGetRoi } from 'libs/queries/extApi/roi';
+import { useMemo } from 'react';
+
+export const useRoi = (strategyId: string) => {
```
[the remark this answers is not in the document]
Comment (zavelevsky) [reply src/hooks/useRoi.ts]: done

File: src/utils/carbonApi.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (zavelevsky) [reply src/utils/carbonApi.ts:65]: dunno?

File: src/components/strategies/overview/strategyBlock/StrategyBlockRoi.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (zavelevsky) [reply src/components/strategies/overview/strategyBlock/StrategyBlockRoi.tsx:16]: animation will be defined in a separate issue by @Costa769 and @ashachaf

**Pull request merged**